convert/convert.h

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file
```

Comment:
Please rename this to `convert/lib.h` -- all libraries have a single header file as an entry point